plugin-server/src/cdp/services/messaging/mailjet-manager.service.test.ts

```diff
@@ -0,0 +1,256 @@
+import crypto from 'crypto'
+import express from 'express'
+
+import { Hub } from '../../../types'
+import { parseJSON } from '../../../utils/json-parse'
+import { MessagingMailjetManagerService } from './mailjet-manager.service'
+
+describe('MessagingMailjetManagerService', () => {
+    let service: MessagingMailjetManagerService
+    let mockHub: Hub
+    const secretKey = 'test-secret-key'
+
+    beforeEach(() => {
+        mockHub = {
+            MAILJET_SECRET_KEY: secretKey,
+        } as Hub
+        service = new MessagingMailjetManagerService(mockHub)
+    })
+
+    describe('handleWebhook', () => {
+        it('should return 403 if required headers are missing', async () => {
+            const req = {
```

Comment:
Pull this out to a function so that its easier to create a dummy request and just modify the bits that should be different